packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.test.js

```diff
@@ -560,18 +561,24 @@ const MOCK_PRODUCT = {
     c_width: 'Z'
 }
 
-test('Renders AddToCartModal', () => {
+test('Renders AddToCartModal with multiple products', () => {
```

Comment:
I tend to make all tests use at least some amount of `await` / `async` / `waitFor` so I would make this an `async` function